GUI/python source/UMSATSGUI_support.py

```diff
@@ -93,78 +95,88 @@ def init(top, gui, *args, **kwargs):
     threadQueue2 = queue.Queue()  # This queue is used to transfer data from the read thread to the gui to be displayed.
     thread2 = None
 
-    date = str(datetime.datetime.today().month) + "-" + str(datetime.datetime.today().day) + "_" + str(datetime.datetime.today().hour) + "h" + str(datetime.datetime.today().minute) + "m"
 
-    serial1 = None
-    serial2 = None
+def clear_display(display1, display2):
 
-def clearDisplay(e,display1,display2):
     print("clearing display")
-    display1.delete("0",END)
-    display2.delete("0",END)
+    display1.delete("0", END)
+    display2.delete("0", END)
+
 
-def updateDisplay(event):
+def update_display(event):
     """ This function prints any new serial data to the display.
     """
+
+    # event.widget is topLevel.
     display1 = event.widget.children['!labelframe'].children['!listbox']
     display2 = event.widget.children['!labelframe'].children['!listbox2']
-    #print("updating display")
+
+    # print("updating display")
+
     global canReceive
+
     if not threadQueue1.empty():
+
         data1 = threadQueue1.get_nowait()  # Get any data from the queue.
-        if LOGGING:logFile1.write(data1) ; # save it to log file
+        if LOGGING:
+            logFile1.write(data1)  # Save it to log file.
+
         data = data1.split(" ")
-        if(len(data)>1):
-            id = data[1]
 
+        # If the length of data is less than 1, then it is invalid.
+        if len(data) > 1:
+            can_id = data[1]
         else:
-            id = ""
-        if((id == 0x100) or (id == 0x200) or (id == 0x300) or (id == 0x400) or (id == 0x500) ):
-            pass;
-        if(id == "0x600"):
-            length = int(data[3])
-            #print("ID: "+ id+ " len: "+ str(length))
-            for index in range(5,length+5):
+            can_id = ""
+
+        # For now ignore any mesasge with ID except Serial_ID.
+        if (can_id == 0x100) or (can_id == 0x200) or (can_id == 0x300) or (can_id == 0x400) or (can_id == 0x500):
```

Comment:
The magic numbers ( the CAN id's) should be moved into separate constants. Also, perhaps you can place them in a global list (e.g. named "ACCEPTED_IDS") so that you can do "if can_id in ACCEPTED_IDS".